File: src/main/java/io/github/coolman4567/andonium/item/ModToolTiers.java
package io.github.coolman4567.andonium.item;

import net.minecraft.world.item.Tier;
import net.minecraft.world.item.crafting.Ingredient;
import net.neoforged.neoforge.common.SimpleTier;
import io.github.coolman4567.andonium.util.ModTags;

public class ModToolTiers {
    public static final Tier ANDONIUM = new SimpleTier(ModTags.Blocks.INCORRECT_FOR_ANDONIUM_TOOL,
            2200, 19f, 0f, 58, () -> Ingredient.of(ModItems.ANDONIUM));
}

File: src/main/java/io/github/coolman4567/andonium/item/ModToolTiers.kt
package io.github.coolman4567.andonium.item

import io.github.coolman4567.andonium.item.ModItems.Andonium
import io.github.coolman4567.andonium.util.ModTags
import net.minecraft.world.item.ToolMaterial
import net.minecraft.world.item.crafting.Ingredient
import net.neoforged.neoforge.common.SimpleTier

object ModToolTiers {
    val ANDONIUM_PICKAXE: ToolMaterial = ToolMaterial(ModTags.Blocks.INCORRECT_FOR_ANDONIUM_TOOL, 3300, 65f, 6f, 58, ModTags.Items.ANDONIUM_REPAIR)
    val ANDONIUM_AXE: ToolMaterial = ToolMaterial(ModTags.Blocks.INCORRECT_FOR_ANDONIUM_TOOL, 3300, 65f, 6f, 58, ModTags.Items.ANDONIUM_REPAIR)
}
